### PR TITLE
Remove assigned scenario

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- add remove scenario in engine

**Related issue(s)**

### CODE REVIEW

File: components/director/internal/domain/label/repository.go [hunk 1]
```diff
@@ -173,6 +173,31 @@ func (r *repository) DeleteByKey(ctx context.Context, tenant string, key string)
 
 	return nil
 }
+func (r *repository) GetRuntimeScenariosWhereRuntimesLabelsMatchSelector(ctx context.Context, tenantID, selectorKey, selectorValue string) ([]model.Label, error) {
```
Comment (pkosiec) [inline components/director/internal/domain/label/repository.go:176]: What's the difference between this method and `GetRuntimeScenariosWhereLabelsMatchSelector`?

File: components/director/internal/domain/scenarioassignment/engine.go [hunk 2]
```diff
@@ -66,16 +70,32 @@ func (e *engine) upsertMergedScenarios(ctx context.Context, labels []model.Label
...
+		}
+		set[output] = struct{}{}
+	}
+
+	return str.MapToSlice(set), nil
 }
 
-func (engine) RemoveAssignedScenarios(in []*model.AutomaticScenarioAssignment) error {
-	// TODO: Implement it
+func (e *engine) scenariosDiff(scenarios []interface{}, diffScenario string) ([]string, error) {
```
Comment (pkosiec) [inline components/director/internal/domain/scenarioassignment/engine.go:88]: this is basically appending new Scenario, right?
IF so:

```suggestion
func (e *engine) appendScenario(scenarios []interface{}, scenario string) ([]string, error) {
```

File: components/director/internal/domain/label/repository.go (hunk 1 above)
Comment (dbadura) [reply components/director/internal/domain/label/repository.go:176]: It's leftover. I will remove it.

File: components/director/internal/domain/scenarioassignment/engine.go (hunk 2 above)
Comment (dbadura) [reply components/director/internal/domain/scenarioassignment/engine.go:88]: Nope, It will remove the scenario.
I will rename it to `removeScenario`

Comment (pkosiec) [reply components/director/internal/domain/scenarioassignment/engine.go]: okay

**Pull request merged**